tracpro/msgs/tasks.py

```diff
@@ -55,40 +60,39 @@ def org_task(self, org):
         from .models import InboxMessage
         from tracpro.contacts.models import Contact
 
-        client = org.get_temba_client()
+        def update_or_create_message(message, org):
+            contact = Contact.objects.filter(uuid=message.contact.uuid).first()
+            # If the contact sync task hasn't gotten this contact yet,
+            # don't get the message yet
+            if contact:
+                InboxMessage.objects.update_or_create(
+                    rapidpro_message_id=message.id,
+                    org=org,
```

Comment:
Not sure why to use the defaults dictionary on all of those, if some of those fields are required (ie contact). Can you explain?